src/model/neuroevolution/individual.py

```diff
@@ -48,10 +46,8 @@ def get_random_bias_index(neural_network_layer: Layer) -> int:
     @staticmethod
     def get_random_bias_indexes(
         neural_network_layer: Layer, indexes_count: int
-    ) -> np.ndarray:
-        return np.random.randint(
-            neural_network_layer.biases.shape[0], size=indexes_count
-        )
+    ) -> Union[Tuple[int, ...], Tuple[np.ndarray, ...]]:
```

Comment:
The returned value is always `Tuple[np.ndarray, ...]`

Reply:
Right, done